karma/makeconf.js

```diff
@@ -29,12 +35,26 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
       if (!browsers) {
         throw new Error('Unknown browser');
       }
+    } else if (isDocker) {
+      browsers = ['ChromeInDocker', 'FirefoxInDocker'];
     } else if (process.platform === 'darwin') {
       browsers = ['ChromeWebRTC', 'ElectronWebRTC', 'FirefoxWebRTC', 'Safari'];
     } else {
       browsers = ['ChromeWebRTC', 'FirefoxWebRTC'];
     }
 
+    const strTestRun = (testRun++).toString();
```

Comment:
@makarandp0 Do we need to append the test run number? I think the combination of browser/bver/topology would be unique enough no? Also, I think we should enable test reporting only for docker container. I don't want to push reports when I run integration tests locally.

Reply:
@makarandp0 Maybe instead of `testRun`, we can use the Travis build number if we have access to it?

Reply:
unfortunately, the combination of browser+bver+topology is not unique, as we have some custom logic to split tests per file to workaround some memory leaks (https://github.com/twilio/twilio-video.js/blob/JSDK-2469_dockerized_tests/scripts/karma.js#L32). 

I considered using date/time stamp instead of the testRun, but decided against it, as I found simple suffixes makes more deterministic and more readable 

Travis build number does not help either, as they are part of the same travis build (btw build is already encoded in the uri as seen in example files below) 

https://s3.amazonaws.com/twilio-video-logs/twilio/twilio-video.js/1848/1848.2/testresults-BROWSER-chrome-BVER-stable-TOPOLOGY-group-2.html
https://s3.amazonaws.com/twilio-video-logs/twilio/twilio-video.js/1848/1848.2/testresults-BROWSER-chrome-BVER-stable-TOPOLOGY-group-1.html

Reply:
about pushing reports - the reports are generated all the time, but pushed to S3 by Travis only for CI runs. They don't get pushed for local runs.